### PR TITLE
Update bookings-in-outlook.md

### PR DESCRIPTION
**Pull request by msbemba**
Added cmds to controlling access to EWS

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10002

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 1]
```diff
@@ -131,8 +131,12 @@ Use the **Get-OrganizationConfig** and **Set-OrganizationConfig** commands to fi
    Set-OrganizationConfig -EwsAllowList @{Remove="MicrosoftOWSPersonalBookings"}
    ```
 
-    - To turn on Bookings with me for your organization, add **MicrosoftOWSPersonalBookings** to **EwsAllowList** by running the following command:  
+    - To turn on Bookings with me for your organization, You must set the **EwsApplicationAccessPolicy** to **EnforceAllowList** and  add **MicrosoftOWSPersonalBookings** to **EwsAllowList** by running the following command:
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-in-outlook.md]:
```suggestion
    - To turn on Bookings with me for your organization, you must set the **EwsApplicationAccessPolicy** to **EnforceAllowList** and add **MicrosoftOWSPersonalBookings** to **EwsAllowList** by running the following command:  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**